VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/participant-waiting-room/tests/participant-waiting-room.component.spec.ts

```diff
@@ -349,6 +343,17 @@ describe('ParticipantWaitingRoomComponent when conference exists', () => {
         expect(component.canStartJoinConsultation).toBeFalsy();
     });
 
+    it('should return false/true if the participant is a quick link observer/participant - canStartJoinConsultation', () => {
+        [
+            [Role.QuickLinkObserver, false],
+            [Role.QuickLinkParticipant, true]
+        ].forEach(([role, expected]) => {
+            component.participant.role = role as Role;
+            component.participant.linked_participants = [];
+            expect(component.canStartJoinConsultation).toBe(expected as boolean);
+        });
+    });
```

Comment:
Doesn't need to be resolved here but in the future it's probably better putting the it statement inside the foreach loop as this will make it clearer where any failures are.

Reply:
Followed an existing pattern.